react-app/src/features/package/index.tsx

```diff
@@ -43,9 +43,13 @@ export const DetailsContent = ({ id }: DetailsContentProps) => {
 
   return (
     <div className="w-full py-1 px-4 lg:px-8">
-      <section id="package_overview" className="block md:flex space-x-0 md:space-x-8">
-        <PackageStatusCard submission={submission} />
-        <PackageActionsCard id={id} submission={submission} />
+      <section id="package_overview" className="flex flex-col sm:flex-row sm:justify-between">
+        <div className="w-full -mb-3 sm:mb-0 sm:w-[45%]">
+          <PackageStatusCard submission={submission} />
+        </div>
+        <div className="w-full -mt-3 text-wrap sm:mt-0 sm:text-nowrap sm:min-w-min sm:w-[45%]">
```

Comment:
I also feel like this `w-[45%]` style can be translate into [`flex-1`](https://tailwindcss.com/docs/flex#basic-example)